client/my-sites/stats/all-time/index.jsx

```diff
@@ -36,16 +36,25 @@ module.exports = React.createClass( {
 
 	render: function() {
 		var bestDay = null,
+			statTabs = [ 'posts', 'views', 'visitors', 'best' ],
+			classSets = {},
 			infoIcon = this.state.showInfo ? 'info' : 'info-outline',
 			valueClass,
 			bestViews,
 			classes;
 
+		statTabs.forEach( function( tabName ) {
+			var tabClassOptions = {};
+			tabClassOptions[ 'module-tab' ] = true;
+			tabClassOptions[ 'is-' + tabName ] = true;
+			classSets[ tabName ] = classNames( tabClassOptions );
+		} );
```

Comment:
Fixed in: https://github.com/Automattic/wp-calypso/commit/8f50f38c3bc14f63527d26f2af30ef6c75529b92